packages/react/src/app-layout/AppLayoutHeader.tsx

```diff
@@ -105,3 +106,13 @@ export function AppLayoutHeader({
 		</Flex>
 	);
 }
+
+/**
+ * Offset the container padding against the AppLayoutHeaderBrand padding to allow for visible focus rings
+ */
+const offsetContainerPadding = Object.fromEntries(
+	Object.entries(tokens.containerPadding).map(([breakpoint, padding]) => [
+		breakpoint,
+		padding - 0.5,
```

Comment:
Must admit I'm confused by this (what is the magic -0.5?) but you've changed the visuals because it's shifted too far left 
<img width="543" alt="image" src="https://github.com/user-attachments/assets/7e8e16dc-8c1c-40f6-88f6-b169ad20bed4">

Reply:
Oh that's odd. 0.5rem of padding was removed from the parent and added to this element to allow the outline some space to render. Maybe I missed adding the 0.5rem back in somewhere.

- [ ] Assign the 0.5rem to a constant
- [ ] Fix the variant with the misalignment